pkg/scheduler/schedule_one_test.go

```diff
@@ -2050,6 +2052,178 @@ func TestSchedulerSchedulePod(t *testing.T) {
 	}
 }
 
+// BenchmarkSchedulePod takes benchmarks of the schedulePod function.
+// Note that this function's purpose is benchmark, not validate the behavior of the function,
+// and it doesn't check the response from the function carefully.
+// The behavior should be validated in the TestSchedulerSchedulePod test.
+func BenchmarkSchedulePod(b *testing.B) {
+	preFilterPlugins := make([]st.RegisterPluginFunc, 20)
+	for i := 0; i < 20; i++ {
```

Comment:
Can we define some fields in the struct of `tests`, e.g.  `plugNum` instead of the hardcode it here, so this testcase could be re-used better for other tuning purpose.